### PR TITLE
Vox version Beanyvox

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Ports TauCeti's Beanyvox from PR: https://github.com/TauCetiStation/TauCetiClassic/pull/8893

As per the discord vote, this won. However, old Vox will remain on the server.

Code will be refactored to be more immersive.

image

image

### Body color
These bad bois come in 5 colors thanks to CandleJaxx
1. DEFAULT(GREEN)
image

2. ALBINO
image

3. BROWN
image

4. DARKTEAL
image

5. YELLOW
image

To change your bodycolor, find the vox bodycolor dropdown box and select your desired color, after this, change your tail selection too.

They also come with new sprite accessories and hair styles, just look for the VOX PRIMALIS label on stuff.

**The sprite accessories are all under the HEAD section in the augments+ tab of the character creator.**

**They also have a snout option**

## To-do:

- [x] Add hair styles
- [x] Add bodycolor selection
- [x] Check all default job clothing works without requiring any clothing changes.
- [x] Resprite all shoes to actually work with these vox
- [x] Make all backpacks fit properly

### Job checklist

- [x] Assistant
- [x] Barber
- [x] Atmos tech
- [x] Bartender
- [x] Blueshield
- [x] Botanist
- [x] Bouncer
- [x] Captain
- [x] Cargo Tech
- [x] Chaplain
- [x] Chemist
- [x] CMO
- [x] CE
- [x] Clown
- [x] Cook
- [x] Corrections Officer
- [x] Curator
- [x] Customs Agent
- [x] Detective
- [x] Customs Guard
- [x] Engineering guard
- [x] Geneticist
- [x] HoP
- [x] HoS
- [x] Janitor
- [x] Lawyer
- [x] Medical Doctor
- [x] Mime
- [x] NT Rep
- [x] Orderly
- [x] Paramedic
- [x] Prisoner
- [x] Psychologist
- [x] Quartermaster
- [x] Research Director
- [x] Roboticist
- [x] Science Guard
- [x] Scientist
- [x] Security Medic
- [x] Security Officer
- [x] Shaft Miner
- [x] Station Engineer
- [x] Vanguard Operative
- [x] Virologist
- [x] Warden

Thanks to CandleJaxx for respriting these things.

Original port authors:
Coiscin
TauCeti
Nebula
Eris

## Changelog

:cl:
add: Vox have been overhauled and resprited!
/:cl:

### CODE REVIEW

**Comment by Zonespace27** [comment]
Based as hell.

**Comment by Tastyfish** [comment]
If you're moving clothing to that modules, remember to zap `modular_skyrat\master_files\icons\mob\clothing\species\vox` as well as the inexplicable `modular_skyrat\master_files\icons\mob\clothing\eyes_vox.dmi` and `modular_skyrat\master_files\icons\mob\clothing\head_vox.dmi` that are just pollution.

**Comment by nikita-chekistov** [comment]
Maybe we should also port their leap ability from Tau?

**Comment by UsadoSmokk** [comment]
Oh no, vox, will fuck in the ass, fuck
IMG_20220511_091957
And fix this
.... And write to all the authors of our PR-s

**Comment by Gandalf2k15** [comment]
> Oh no, vox, will fuck in the ass, fuck IMG_20220511_091957 And fix this .... And write to all the authors of our PR-s

That is for greyscale hair so you can pick colore

**Comment by UsadoSmokk** [comment]
> > О нет, вокс, буду трахаться в задницу, трахаться IMG_20220511_091957 И исправлять это.... И пишите всем авторам наших PR-s
> 
> Это для седых волос, чтобы вы могли выбрать цвет

image
about this

**Comment by CandleJaxx** [comment]
> about this

that's been fixed already, gandalf just hasn't updated the pr :p

**Comment by MistakeNot4892** [comment]
For reference, the base Vox icon is from Nebula, not Tau Ceti.

**Comment by Nerev4r** [comment]
Currently the actual race itself is invisible.

**Comment by CandleJaxx** [comment]
WOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

YEAHHHH BABYYYYY

**Comment by Sy-zz-nthe-zz-ticF-zz-ox** [comment]
The change to `modular_skyrat/master_files/code/modules/client/preferences/species_features/digitigrade_legs.dm` seems to have made it so that if a species does not have digitigrade legs without mismatched parts turned on, turning it on does not give them the option, which affects a LOT of species unrelated

**Comment by HalcyonicWolf** [comment]
It seems that the Vox are illiterate by default. Though I'm not 100% sure.

**Comment by HalcyonicWolf** [comment]
Another find: Their survival box comes with an oxygen tank rather than a nitrogen one.

**Comment by VeteranKyl** [comment]
This might of broken some parts of old vox, including spine options. https://github.com/Skyrat-SS13/Skyrat-tg/issues/14308

**Comment by CandleJaxx** [comment]
vox primalis are using v1 vox jumpsuit sprites, not v4

**Comment by Gandalf2k15** [comment]
Merging so people can make changes

File: modular_skyrat/modules/modular_items/lewd_items/code/arousal_system.dm [hunk 1]
```diff
@@ -762,7 +762,7 @@
 			cumface.icon_state = "cumface_lizard"
 		else if(target.dna.species.id == SPECIES_MONKEY)
 			cumface.icon_state = "cumface_monkey"
-		else if(target.dna.species.id == SPECIES_VOX)
+		else if(target.dna.species.id == SPECIES_VOX || target.dna.species.id == SPECIES_VOX_PRIMALIS)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/lewd_items/code/arousal_system.dm:765]: This probably could've used `isvox()` and `isvoxprimalis()`

**Pull request merged**